docs/CPP_STYLE.md

```diff
@@ -4,15 +4,15 @@ In general, [Google's coding standard](https://google.github.io/styleguide/cppgu
 
 Below are our specific (but not all!) exceptions to the Google's coding standard:
 
-- All C++ code should conform to the C++17 standard.
+- All C++ code should conform to the c++20 standard.
```

Comment:
Let's give a respect to our fav language :) Write with capital C++20 here and below.

Reply:
fixed :)